tests/integration/Owncloud/OcisPhpSdk/OcisTest.php

```diff
@@ -30,6 +30,88 @@ public function testServiceUrlTrailingSlash(): void
         $this->assertTrue((is_array($drives) && count($drives) > 1));
     }
 
+    public function testGetAllDrives(): void
+    {
+        $ocis = $this->getOcis('admin', 'admin');
+        $sportDrive = $ocis->createDrive('Sport Project Drive');
+        $this->createdDrives[] = $sportDrive->getId();
+        $managementDrive = $ocis->createDrive('Management Project Drive');
+        $this->createdDrives[] = $managementDrive->getId();
+        $drives = $ocis->getAllDrives();
+        $this->assertCount(4, $drives);
+        foreach ($drives as $drive) {
+            $this->assertInstanceOf(Drive::class, $drive);
+            $this->assertIsString($drive->getId());
+            $this->assertIsString($drive->getName());
+            $this->assertIsObject($drive->getType());
+            $this->assertIsObject($drive->getRoot());
+        }
+        $this->assertTrue([$managementDrive->getName(),$sportDrive->getName()] === [$drives[1]->getName(),$drives[3]->getName()]);
```

Comment:
maybe easier to read if we would use `logicalOr` as done in
https://github.com/owncloud/ocis-php-sdk/blob/d5c6abfe6c1fb0ac2b078d237e7032d30f403ca1/tests/integration/Owncloud/OcisPhpSdk/OcisResourceTest.php#L58-L64

same for other places.